fabric-extensibility-api-v1/src/testmod/java/net/fabricmc/fabric/test/extensibility/FabricCrossbowTests.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.test.extensibility;
+
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.item.ArrowItem;
+import net.minecraft.item.CrossbowItem;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemGroup;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.registry.Registry;
+import net.minecraft.entity.projectile.PersistentProjectileEntity;
+
+import net.fabricmc.api.ModInitializer;
+import net.fabricmc.fabric.api.extensibility.item.v1.FabricCrossbowHooks;
+
+public class FabricCrossbowTests implements ModInitializer {
```

Comment:
Maybe rename this from FabricCrossbowTests to FabricBowApiTestmod? Not sure, someone else should help brainstorm names, I just know I don't like FabricCrossbowTests.

Reply:
I think one class per different item is better, mostly because people can look for it with ease, rather than have 6 different pieces of the api in their face